org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/formatter/FormatterRegressionTests.java

```diff
@@ -16536,4 +16536,357 @@ public int sample(String param1) {
 				""";
 		formatSource(input, expected);
 	}
+
+	public void testMarkdownUnorderedListTags() throws JavaModelException {
+		setComplianceLevel(CompilerOptions.VERSION_23);
+		String input = """
+				/// ## Unordered Lists
+				/// - Item 1
+				///   - Subitem  2.1
+				///     - Subitem   2.2
+				///   -  Item3
+				/// -  sdd
+				class Mark {
+				}
+				""";
+		String expected = """
+				/// ## Unordered Lists
+				///   - Item 1
```

Comment:
this should not be indented like this, so probably `indentToSet = 2 * (currentLevel - 1);`